src/app/producer-shelf/components/tv2-action-card/tv2-action-card.component.html

```diff
@@ -1 +1,3 @@
-{{ action.name }}
+<span class="pt-1 ps-2 pe-2">{{ action.name }}</span>
+<div #titleContainer class="titleContainer"></div>
```

Comment:
Suggestion: The newly introduced element can be avoided by making the host element display as inline-flex and use z-index of 1 with a visible vertical overflow to let the text overflow over the image.
So the HTML would be
```html
<div #titleContainer class="title pt-1 ps-2 pe-2">{{ action.name }}</div>
<img *ngIf="isVideoClipAction() && videoClipThumbnailUrl" [ngSrc]="videoClipThumbnailUrl" [width]="getThumbnailWidth()" [height]="getThumbnailHeight()" [alt]="action.name" top>
```
and the SCSS
```scss
:host {
  display: inline-flex;
  flex-direction: column;

  .title {
    height: 25px;
    overflow-y: visible;
    z-index: 1;
  }
}

Reply:
The newly introduced element has been removed again.
However, I changed `overflow-y: visible` with `position: relative` since `z-index` wouldn't be applied without it:
```
.title {
  position: relative;
  height: 25px;
  z-index: 1;
}
```

Sidenote: I might be wrong here, but `flex-direction` only works if you are using `display: flex`?